_posts/en/newsletters/2025-07-25-newsletter.md

```diff
@@ -0,0 +1,183 @@
+---
+title: 'Bitcoin Optech Newsletter #364'
+permalink: /en/newsletters/2025/07/25/
+name: 2025-07-25-newsletter
+slug: 2025-07-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **LND gossip filter DoS vulnerability:** Matt Morehouse
+  [posted][morehouse gosvuln] to Delving Bitcoin about a vulnerability
+  affecting past versions of LND that he previously [responsibly
+  disclosed][topic responsible disclosures].  An attacker could
+  repeatedly request historic gossip messages from an LND node until it
+  ran out of memory and was terminated.  The vulnerability was fixed in
+  LND 0.18.3, released September 2024.
+
+- **Chain code witholding for multisig scripts:** Jurvis Tan
+  [posted][tan ccw] to Delving Bitcoin about research he performed with
+  Jesse Posner into improving the privacy and security of multisig
+  collaborative custody.  In a typical collaborative custody service, a
+  2-of-3 multisig may be used with the three keys being:
+
+  - A user _hot key_ that is stored on a networked device and signs
+    transactions for the user (either manually or through software
+    automation).
+
+  - A provider hot key that is stored on a separate networked device
+    under the exclusive control of the provider.  The key only signs
+    transactions according to a policy previously defined by the user,
+    such as only allowing spending up to _x_ BTC a day.
+
+  - A user _cold key_ that is stored offline and is only used if either
+    the user hot key is lost or the provider ceases signing authorized
```

Comment:
```suggestion
  - A user's _cold key_ that is stored offline and is only used if either
    the hot key is lost, or the provider ceases signing authorized
```
Or "the user's hot key" (but I think it's clear it's the user's from earlier in the sentence).

Reply:
Leaving as-is.